.gitmodules

```diff
@@ -1,3 +1,6 @@
+[submodule "components/vendor/kubeflow-manifests"]
+	path = components/vendor/kubeflow-manifests
+	url = git@github.com:kubeflow/manifests.git
```

Comment:
Let's either have Git or HTTPS URLs for the submodules, but not both at the same time. I would prefer HTTPS URLs, as one doesn't have to add any SSH certs in Github to be able to use them.

Reply:
That was a typo, but thanks